in.varb.Ausweiskopie.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: "in.varb.Ausweiskopie"
+runtime: "org.freedesktop.Platform"
+runtime-version: "24.08"
+sdk: "org.freedesktop.Sdk"
+command: ausweiskopie
+finish-args:
+  - --socket=x11
+  - --share=ipc
+modules:
+  # Step 1: Install a current version of Tcl/Tk
+  - name: "tcl"
+    sources:
+      - type: archive
+        url: http://prdownloads.sourceforge.net/tcl/tcl8.6.16-src.tar.gz
+        sha256: 91cb8fa61771c63c262efb553059b7c7ad6757afa5857af6265e4b0bdc2a14a5
+        x-checker-data:
+          type: html
+          url: "https://sourceforge.net/projects/tcl/rss"
+          version-pattern: "https://sourceforge.net/.+/tcl(8\\.6\\.[\\d\\.]*\\d)-src.tar.gz/download"
+          url-template: "http://prdownloads.sourceforge.net/tcl/tcl$version-src.tar.gz"
+    subdir: unix
+    post-install:
+      - "chmod +w ${FLATPAK_DEST}/lib/libtcl8.6.so"
+  - name: "tk"
+    sources:
+      - type: archive
+        url: http://prdownloads.sourceforge.net/tcl/tk8.6.16-src.tar.gz
+        sha256: be9f94d3575d4b3099d84bc3c10de8994df2d7aa405208173c709cc404a7e5fe
+        x-checker-data:
+          type: html
+          url: "https://sourceforge.net/projects/tcl/rss"
+          version-pattern: "https://sourceforge.net/.+/tk(8\\.6\\.[\\d\\.]*\\d)-src.tar.gz/download"
+          url-template: "http://prdownloads.sourceforge.net/tcl/tk$version-src.tar.gz"
+    subdir: unix
+    post-install:
+      - "chmod +w ${FLATPAK_DEST}/lib/libtk8.6.so"
+
+  # Step 2.1: Install requirements for requirements...
+  - name: "dbus"
+    buildsystem: meson
+    build-options:
+      config-opts:
+        - "-Dsystemd_system_unitdir=/tmp"
+        - "-Dsystemd_user_unitdir=/tmp"
+    sources:
+      - type: archive
+        url: https://dbus.freedesktop.org/releases/dbus/dbus-1.16.2.tar.xz
+        sha256: 0ba2a1a4b16afe7bceb2c07e9ce99a8c2c3508e5dec290dbb643384bd6beb7e2
+        x-checker-data:
+          type: html
+          url: "https://dbus.freedesktop.org/releases/dbus/"
+          version-pattern: "dbus-([\\d\\.]+\\d).tar.xz"
+          url-template: "https://dbus.freedesktop.org/releases/dbus/dbus-$version.tar.xz"
+  - name: "patchelf"
+    sources:
```

Comment:
patchelf is needed at runtime? If not add a `cleanup: ["*"]`

Reply:
I will look into cleanup, likewise for freedesktop dbus.